hooks/pre_commit/05-puppet_certs_exist.rb

```diff
@@ -8,8 +8,8 @@
 
 if puppet_ca_enabled && key_path && File.exist?(key_path) && !kafo.skip_checks_i_know_better?
   if !puppet_ca_cert.empty? && !File.exist?(puppet_ca_cert)
-    fail_and_exit("The file #{puppet_ca_cert} does not exist.\n #{client_message}\n" \
-    " - if you use custom Puppet SSL directory (--foreman-proxy-ssldir) make sure the directory exists and contain the CA certificate.\n", 101)
+    fail_and_exit("The file #{puppet_ca_cert} does not exist.\n #{client_message}\n " \
+                  "- if you use custom Puppet SSL directory (--foreman-proxy-ssldir) make sure the directory exists and contain the CA certificate.\n", 101)
```

Comment:
```suggestion
    fail_and_exit("The file #{puppet_ca_cert} does not exist.\n #{client_message}\n" \
                  " - if you use custom Puppet SSL directory (--foreman-proxy-ssldir) make sure the directory exists and contain the CA certificate.\n", 101)
```

Reply:
Making these changes, raise two new offenses Layout/LineEndStringConcatenationIndentation and Layout/LineContinuationLeadingSpace. Should I disable it inline? @ekohl